patcher.cpp

```diff
@@ -26,6 +27,24 @@ static s32 GetWC24Error(s32 wc24_fd, NWC24Config& config)
   return reinterpret_cast<s32 *>(io_buf)[2];
 }
 
+static s32 GetSystemMenuIOS() {
+  s32 ret;
+  u32 view_size{};
+
+  ret = ES_GetTMDViewSize(0x100000002LL, &view_size);
+  if (ret < 0)
+    return ret;
+
+  std::array<u8, view_size> _buffer{};
```

Comment:
You will need to use a vector instead, array doesn't like that `view_size` is not const

```cpp
  std::vector<u8> _buffer(view_size);
  tmd_view* tmd = reinterpret_cast<tmd_view*>(_buffer.data());
```